src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -43,7 +43,6 @@ final class ExposureNotificationSettingViewController: UITableViewController, Ac
 
 		tableView.sectionFooterHeight = 0.0
 		tableView.separatorStyle = .none
-		tableView.estimatedRowHeight = 1000 // this is need to prevent jumping of scroll position when tableview datasource is reloaded
```

Comment:
We should test the scenario mention in the comment why this line was added.

Reply:
Alle tests on my side with iOS 14 and 15 was successful. @30mar also tested 12.5 successfully.
Why the line was added therefore ... don't know why. All UI scrolls looks fine at the moment. 
Is that okay for you?

Reply:
Yes. Thanks